jicoco/src/main/kotlin/org/jitsi/rest/JettyBundleActivatorConfig.kt

```diff
@@ -95,6 +95,32 @@ class JettyBundleActivatorConfig(
         "default" { true }
     }
 
+    val tlsProtocols: List<String> by config {
+        "$newPropertyPrefix.tls-protocols".from(JitsiConfig.newConfig)
+        "default" { DEFAULT_TLS_PROTOCOLS }
+    }
+
+    val tlsCipherSuites: List<String> by config {
+        "$newPropertyPrefix.tls-cipher-suites".from(JitsiConfig.newConfig)
+        "default" { DEFAULT_TLS_CIPHER_SUITES }
+    }
+
     override fun toString() = "host=$host, port=$port, tlsPort=$tlsPort, isTls=$isTls, keyStorePath=$keyStorePath, " +
         "sendServerVersion=$sendServerVersion"
```

Comment:
Do you want to include the new variables `tlsProtocols` and `tlsCipherSuites` into `toString` function here?